dashboard/modules/stats_collector/stats_collector_head.py

```diff
@@ -233,9 +233,11 @@ def _process_actor_table_data(data):
                 # Update actors.
                 DataSource.actors[actor_id] = actor_table_data
                 # Update node actors.
-                node_actors = dict(DataSource.node_actors.get(node_id, {}))
-                node_actors[actor_id] = actor_table_data
-                DataSource.node_actors[node_id] = node_actors
+                if node_id != "f" * 56:
```

Comment:
It's better to set a const to `stats_collector_consts.py`
```python
NIL_NODE_ID = ray.NodeID.nil().hex()
```
then check `if node_id != stats_collector_consts.NIL_NODE_ID`.

Reply:
Sounds like a good idea!

Reply:
Can you add this check to `GetAllActorInfo` processing logic, too? Then, the assertion could be added to the `test_stats_collector.py`:
```python
response = requests.get(webui_url + "/test/dump?key=node_actors")
response.raise_for_status()
result = response.json()
assert stats_collector_consts.NIL_NODE_ID not in result["data"]["node_actors"]
```